apps/jobs/views.py

```diff
@@ -1589,7 +1597,15 @@ def update_partially_evaluated_submission(request, challenge_pk):
         submission_pk = request.data.get("submission")
         submission_status = request.data.get("submission_status", "").lower()
         job_name = request.data.get("job_name", "").lower()
-        submission = get_submission_model(submission_pk)
+        try:
+            submission = get_submission_model(submission_pk)
+        except NotFound:
+            response_data = {
+                "error": "Submission does not exist with submission_id {}".format(
```

Comment:
Change `Submission does not exist with submission_id {}` to `Submission with submission_pk {} does not exist`